src/exchange/exchange.ts

```diff
@@ -17,72 +24,113 @@ export const devtoolsExchange: Exchange = ({ client, forward }) => {
       );
   }
 
+  // Expose graphql url for introspection
   window.__urql__ = {
```

Comment:
This can become { url: client.url }

Reply:
This is fine - if we change the window object, we break compatibility with the current devtools extension release